src/menu.cpp

```diff
@@ -107,37 +168,43 @@ void MenuBody::paint(BitmapBuffer * dc)
 {
   dc->clear(COLOR_THEME_PRIMARY2);
 
-  for (unsigned i = 0; i < lines.size(); i++) {
+  int y = 0;
+  for (int i = 0; i < lines.size(); i++) {
     auto& line = lines[i];
     LcdFlags flags = COLOR_THEME_SECONDARY1 | MENU_FONT;
-    if (selectedIndex == (int)i) {
+
+    // draw selection if appropriate
+    if (selectedIndex == i && ! line.isSeparator) {
       flags = COLOR_THEME_PRIMARY2 | MENU_FONT;
       if (COLOR_THEME_FOCUS != COLOR_THEME_PRIMARY2) {
-        dc->drawSolidFilledRect(0, i * MENUS_LINE_HEIGHT, width(),
+        dc->drawSolidFilledRect(0, y, width(),
                                 MENUS_LINE_HEIGHT, COLOR_THEME_FOCUS);
       }
     }
-    if (line.drawLine) {
-      line.drawLine(dc, 0, i * MENUS_LINE_HEIGHT, flags);
+
+    if (line.isSeparator) {
+      dc->drawHorizontalLine(5, y + MENUS_SEPARATOR_HEIGHT / 2, width() - 10, 255, COLOR_THEME_SECONDARY2);
```

Comment:
Where is `MENUS_SEPARATOR_HEIGHT ` defined?

Reply:
in libopenui_config.h under horus targets.